ymer/src/main/java/com/avanza/ymer/PersistedInstanceIdRecalculationService.java

```diff
@@ -32,6 +34,7 @@
 import com.mongodb.client.model.IndexOptions;
 
 public class PersistedInstanceIdRecalculationService implements PersistedInstanceIdRecalculationServiceMBean {
+	private static final int BATCH_SIZE = 1_000;
```

Comment:
Is there a calculation behind choosing `1_000` here? Why not, for example `5_000`, `10_000` or even larger?

Reply:
1000 is just an approximation from previous experience, there is no hard calculation behind it. I expect the performance gains to decrease with larger sizes. Moving from 1 to 10 would make a big increase, from 10 to 100 a slightly smaller increase on top of that, from 100 to 1000 a slightly smaller increase yet. 5 or 10000 probably works well, maybe there is some more performance to be gained.
Larger batch sizes increases the risk somewhat, so there is no point in making it too large. 
It should probably be configurable rather than hardcoded.